docs/commands_states.rst

```diff
@@ -32,25 +44,26 @@ installed code.  Note that a key design feature is that is it straightforward fo
 to implement their own states, often with just a few lines of code.  See the `User-defined states`_
 section for details.
 
-Commands archive
-----------------
+Commands archive v1
+-------------------
 
 The basic way to select commands is with the |get_cmds| method.  For example you can find
 load commands from early in 2013 with::
 
   >>> from kadi import commands
   >>> cmds = commands.get_cmds('2013:001:00:00:00', '2013:001:00:56:10')
   >>> print(cmds)
-           date            type      tlmsid   scs step timeline_id params
-  --------------------- ---------- ---------- --- ---- ----------- ------
-  2013:001:00:37:37.653   ORBPOINT       None   0    0   426098988    N/A
-  2013:001:00:53:07.181 COMMAND_SW   AOACRSTD 129 1524   426098990    N/A
-  2013:001:00:54:07.181 COMMAND_SW   AOFUNCDS 129 1526   426098990    N/A
-  2013:001:00:55:07.181 COMMAND_SW   AOFUNCDS 129 1528   426098990    N/A
-  2013:001:00:56:07.181 COMMAND_SW   AONMMODE 129 1530   426098990    N/A
-  2013:001:00:56:07.181    ACISPKT AA00000000 132 1620   426098991    N/A
-  2013:001:00:56:07.181   SIMTRANS       None 132 1623   426098991    N/A
-  2013:001:00:56:07.438 COMMAND_SW   AONM2NPE 129 1532   426098990    N/A
+          date            type      tlmsid   scs step      time     timeline_id   vcdu  params
```

Comment:
It looks like the other examples previously didn't print with a "time" column as well, but that doesn't match the flight output.

Reply:
Yeah, I got a little lazy here and didn't update the rest of the `v1` examples outputs since `v1` will be deprecated and then removed.